src/components/constant.tsx

```diff
@@ -0,0 +1,5 @@
+export const API = 'http://localhost:1337/api';
+export const AUTH_TOKEN = 'authToken';
+export const BEARER = 'Bearer';
```

Comment:
The constant folder should be a stand-alone folder instead of been inside component folder